packages/mdc-radio/index.ts

```diff
@@ -22,106 +22,97 @@
  */
 
 import MDCComponent from '@material/base/component';
-/* eslint-disable no-unused-vars */
-import {MDCSelectionControlState, MDCSelectionControl} from '@material/selection-control/index';
-/* eslint-enable no-unused-vars */
-import MDCRadioFoundation from './foundation';
+import {EventType, SpecificEventListener} from '@material/dom/index';
+import {RippleCapableSurface} from '@material/ripple/index';
 import {MDCRipple, MDCRippleFoundation} from '@material/ripple/index';
+import {MDCSelectionControl} from '@material/selection-control/index';
 
-/**
- * @extends MDCComponent<!MDCRadioFoundation>
- * @implements {MDCSelectionControl}
- */
-class MDCRadio extends MDCComponent {
-  static attachTo(root) {
+import MDCRadioFoundation from './foundation';
+
+class MDCRadio extends MDCComponent<MDCRadioFoundation>
+  implements RippleCapableSurface, MDCSelectionControl {
+
+  static attachTo(root: HTMLElement) {
     return new MDCRadio(root);
   }
 
-  /** @return {boolean} */
-  get checked() {
-    return this.nativeControl_.checked;
+  root_!: Element;
```

Comment:
Add a comment explaining why we're overriding this property from `MDCComponent` with public visibility (i.e., for `RippleCapableSurface`).

Reply:
Done